Added file feature for base64 cipher function

Sedatyf opened the pull request:
You can now choose between a string or file for Base64 encryption/decryption. 
Check how I made it @SoftAddict and tell me if you want to keep it as is or not

Sedatyf committed:
Added file feature for base64 cipher function

Cyb0rgbytes commented on Decipher-Framework.py:
```diff
@@ -33,14 +41,36 @@ def decipher_base64():
+            print(f'\n{Fore.WHITE}Your Ecrypted Text >>> {base64.b64encode(string_input.encode()).decode()}')
         elif user_input == 'd':
-            print('Please enter the string you would like Base64 decoded')
-            string_input = input('String: ')
```
I Would like to keep this prompt so user has a way of knowing what to type and the availability of a prompt makes it better and readable by the user i think. What do you think?

Cyb0rgbytes committed:
Update Decipher-Framework.py

Updated and merged from Sedatyf branch
Thanks to Sedatyf we managed to add a file attach ability to the encrypter and decrypter which improve usability.
Added : 
- File based Functions and strings.
- Other improvements.
